code/game/gamemodes/nuclear/nuclear_challenge.dm

```diff
@@ -52,7 +54,7 @@
 
 	var/obj/item/device/radio/uplink/U = new /obj/item/device/radio/uplink(get_turf(user))
 	U.hidden_uplink.uplink_owner= "[user.key]"
-	U.hidden_uplink.uses = 280
+	U.hidden_uplink.uses = CHALLENGE_TELECRYSTALS + (((player_list.len + CHALLENGE_MIN_PLAYERS) * 4) * CHALLENGE_SCALE_FACTOR) // Add 4 telecrystals per player above CHLALENGE_MIN_PLAYERS (50). Scale factor multiply it.
```

Comment:
`player_list.len + CHALLENGE_MIN_PLAYERS` is wrong.

First off, let's say player_list.len equals 100. You then add 50, for a total of 150. You then are multiplying this by 4 and adding that number of TC to the ops baseline of 280 from challenge mode.

You just gave them 880 TC.

You probably meant to SUBTRACT the CHALLENGE_MIN_PLAYERS from the player_list.len, so you actually only count the players after 50 as qualifying for bonus TC.

Reply:
You also should probably make that `4` a define rather than a magic number, and consider dropping the CHALLENGE_SCALE_FACTOR define or adjust how it is used.

For example, you could use that to actually be the multiple of players needed to gain additional TC. For example, you could rewrite this to be `round((players_list.len - CHALLENGE_MIN_PLAYERS)/CHALLENGE_SCALE_PLAYERS) * CHALLENGE_SCALE_BONUS` which takes the total players, subtracts the minimum number to declare war (50), then divides the excess by the scale factor before multiplying that by the bonus TC for scaling.

To better illustrate this, let's make the following assumptions
- players_list.len = 100
- CHALLENGE_MIN_PLAYERS = 50
- CHALLENGE_SCALE_PLAYERS = 3 
- CHALLENGE_SCALE_BONUS = 4
With these assumptions, we get the following
`round((100 - 50)/3) * 4` which becomes `round(50/3) * 4` and then `16*4` and finally we reach the end result of 64.

This gives you better control over the scaling, as you can tweak the multiple of players in excess of the minimum to qualify for bonus as well as the bonus to give for each multiple. It also rounds so you don't need to worry about fractional TC or such

Reply:
Adding challenge_min_players instead of subtract is an accident I didn't intend. I also believe your implementation to be better and will change it.

Reply:
If you need help, hit me up on Discord later when I'm home and I can check your math and such.

Reply:
Will do. The commit will be up very soon after I test and see if it run properly.